.github/workflows/package.yml

```diff
@@ -237,6 +237,7 @@ jobs:
     env:
       APACHE_ARROW_REPOSITORY: ${{ github.workspace }}/apache-arrow
     runs-on: ubuntu-latest
+    timeout-minutes: ${{ (matrix.id == 'debian-trixie-arm64' && 270) || 90 }}
```

Comment:
I suppose that The timeout of packages for amd64 is too long.
Probably, a build of packages for amd64 finish within 60 min in most cases.

Reply:
Currently, we are only splitting Debian trixie because it takes longer, without separating `arm64` (`aarch64`) and `amd64` (`x86_64`). Looking at the following, Debian bookworm `arm64` is taking 80 minutes. So I think the current setting isn't bad.
- https://github.com/groonga/groonga/actions/runs/10481431201/usage

Would it be better to separate `arm64` and `amd64`?

Reply:
Ah, I see.

Probably, Ubuntu focal arm64 is slow same as Debian trixie arm64.
So, I suppose that it would be better to separate `arm64` and `amd64`.